setup.cfg

```diff
@@ -0,0 +1,37 @@
+[metadata]
+name = pangaeapy
+version = 1.0.17
+author = Robert Huber
+author_email = rhuber@uni-bremen.de
+description = This module allows to download and analyse metadata as well as data from tabular PANGAEA (https://www.pangaea.de) datasets.                  Usage:         import pangaeapy.pandataset as pd         ds= pd.PanDataSet(787140)         print(ds.title)         print(ds.data.head())                  Please visit the github project page to see more documentation and some examples:         https://github.com/pangaea-data-publisher/pangaeapy
+long_description = file: README.md
```

Comment:
This will show the contents of your README on https://pypi.org/project/pangaeapy/ in the description field.